tests/Unit/Domain/CoordinateMaps/TestMapHelpers.hpp

```diff
@@ -101,6 +102,34 @@ void test_jacobian(const Map& map,
   }
 }
 
+template <typename Map>
+void test_jacobian(
+    const Map& map, const std::array<double, Map::dim>& test_point,
+    const double time,
+    const std::unordered_map<
+        std::string, std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>>&
+        functions_of_time) noexcept {
+  // Our default approx value is too stringent for this test
+  const auto compute_map_point =
+      [&map, time,
+       &functions_of_time](const std::array<double, Map::dim>& point) noexcept {
+        return map(point, time, functions_of_time);
+      };
+  Approx local_approx = Approx::custom().epsilon(1e-10).scale(1.0);
+  const double dx = 1e-4;
```

Comment:
Why this particular value?

Reply:
Because it works :man_shrugging: This has been in the code for months or years.